Lib/test/test_defaultdict.py

```diff
@@ -183,5 +183,30 @@ def test_pickling(self):
             o = pickle.loads(s)
             self.assertEqual(d, o)
 
+    def test_union(self):
```

Comment:
I see no tests for ‘|=‘?

Reply:
defaultdict inherits a perfectly fine `|=` from `dict`.

I can add tests for it here if you'd like, but the tests for `dict` already have the same coverage.

Reply:
D'oh. Maybe add a comment to this effect?